tooling/compare-builds-postfunctions.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+# ********************************************************************************
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
```

Comment:
Copyright 2025....

Reply:
<Year Creation>[, Year Modified]

Reply:
Thanx!  From curiosity (as it is off topic)  what is it good for? Usually all those are in git log... TY!

Reply:
Well depends on what source... but it is relevant to Licensed source and various Corporate requirements and guidelines, but also just good practice